app/views/accounts/_webauthn.html.slim

```diff
@@ -0,0 +1,16 @@
+.clearfix.border-top.border-blue-light
+.clearfix.border-top.border-blue-light
+  .p2.col.col-12.border-bottom.border-blue-light
```

Comment:
I found that putting this here causes a double line when there's no configuration in the phone list. I moved it to be a `.border-top.border-blue-light` on the classes in line 10 (the top of the configuration)